api-tests/dev_apis/crypto/test_c043/test_c043.c

```diff
@@ -108,6 +108,10 @@ int32_t psa_key_agreement_test(security_t caller)
         /* Abort a generator */
```

Comment:
The key should also be destroyed when expected status of CHECKPOINT 6 is not PSA_SUCCESS.
So, it must also be added inside if condition in line 84

Reply:
Fixed, can you please re-review?